detox/test/detox.config.js

```diff
@@ -99,7 +99,12 @@ module.exports = {
       type: "android.emulator",
       device: {
         avdName: "Pixel_API_28"
-      }
+      },
+      launchArgs: {
```

Comment:
This is JavaScript, so I'd advise reusing a single `launchArgs` object across all the configurations.